### PR TITLE
webhooks: move GraphQL resolver to enterprise package.

### PR DESCRIPTION
**Pull request by sashaostrikov**
This PR completes https://github.com/sourcegraph/sourcegraph/issues/43656.

Moving gql resolvers under enterprise package.

Test plan:
Existing test cases should pass after refactoring.

### CODE REVIEW

File: cmd/frontend/graphqlbackend/executor_secrets_test.go [hunk 1]
```diff
@@ -573,7 +573,7 @@ func TestExecutorSecretsIntegration(t *testing.T) {
 		t.Fatal("invalid number of access logs found in DB")
 	}
 
-	s, err := NewSchema(db, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil)
+	s, err := NewSchema(db, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil)
```
Comment (ryanslade) [inline cmd/frontend/graphqlbackend/executor_secrets_test.go:576]: lol

Comment (ryanslade) [reply cmd/frontend/graphqlbackend/executor_secrets_test.go:576]: This is the kind of code we don't show the Go haters

**Review by ryanslade — APPROVED** [review]
LGTM assuming you've done some manual testing to confirm it still works

**Comment by sashaostrikov** [comment]
> LGTM assuming you've done some manual testing to confirm it still works

yep, I went to webhooks page, which used the resolvers that I've moved 👍🏻

File: cmd/frontend/graphqlbackend/executor_secrets_test.go (hunk 1 above)
Comment (mrnugget) [reply cmd/frontend/graphqlbackend/executor_secrets_test.go:576]: Which is why I've introduced these helpers 😬 

https://github.com/sourcegraph/sourcegraph/blob/71e7f4282e6090a704d055a1379a856c37e1d98b/cmd/frontend/graphqlbackend/graphqlbackend.go#L314-L332

Comment (mrnugget) [reply cmd/frontend/graphqlbackend/executor_secrets_test.go:576]: https://github.com/sourcegraph/sourcegraph/pull/44145

**Review by mrnugget — APPROVED** [review]

**Pull request merged**